PipelinesTasks/PRMetrics/src/LICENSE.txt

```diff
@@ -1368,7 +1368,7 @@ THE SOFTWARE.
 
```

Comment:
nit: not apart of this PR, but how come the copyright comment is in twice around line 49/50, is that on purpose?

Reply:
I realise this is strange, but the file is autogenerated and it's seemingly correct. The second instance of the copyright statement is coming from <https://github.com/microsoft/tslib/blob/main/LICENSE.txt>. Not sure where the first comes from, but I assume it is some repo metadata. I guess they're both the same in this case, causing the repetition.